# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../cron/syslog_atrun.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./misc.c.orig	Sun Jun 10 14:55:07 2001
+++ ./misc.c	Sun Jun 10 14:54:32 2001
@@ -531,8 +531,11 @@
 	openlog(ProgramName, LOG_PID);
 # endif
 	  
-	syslog(LOG_INFO, "(%s) %s (%s)", username, event, detail);
-
+ 	/* HACK: Don't create syslog messages for atrun */
+ 	if ( strcmp(username, "root") || strcmp(event, "CMD") ||
+ 	     strcmp(detail, "test -x /usr/lib/atrun && /usr/lib/atrun") )
+ 		syslog(LOG_INFO, "(%s) %s (%s)\n", username, event, detail);
+  
 	closelog();
 #endif /*SYSLOG*/
 
